libraries/api-extractor-model/src/model/TypeParameter.ts

```diff
@@ -78,12 +79,18 @@ export class TypeParameter {
    */
   public name: string;
 
+  /**
+   * Whether the type parameter is optional. True IFF there exists a `defaultTypeExcerpt`.
+   */
+  public isOptional: boolean;
```

Comment:
It seems like knowing what the default type is would also be relevant to documentation. If the constraint is something simpleish like `extends HTMLElement` and the default type is something pretty specific like `HTMLCanvasElement`, that's worth calling out in documentation.

Reply:
Actually nevermind, looks like that information is already there.